packages/flutter/test/material/text_field_focus_test.dart

```diff
@@ -548,6 +563,63 @@ void main() {
     expect(focusNodeA.hasFocus, false);
     expect(focusNodeB.hasFocus, true);
   }, variant: TargetPlatformVariant.desktop());
+
+  testWidgets('TextFormField gets blurred when tapping outside of the widget on mobile browsers', (WidgetTester tester) async {
+    final FocusNode focusNodeA = FocusNode();
+    final FocusNode focusNodeB = FocusNode();
+    final Key key = UniqueKey();
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: ListView(
+            children: <Widget>[
+              TextField(
+                focusNode: focusNodeA,
+              ),
+              Container(
+                key: key,
+                height: 200,
+              ),
+              TextField(
+                focusNode: focusNodeB,
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+
+    final TestGesture down1 = await tester.startGesture(tester.getCenter(find.byType(TextField).first));
+    await tester.pump();
+    await tester.pumpAndSettle();
+    await down1.up();
+    await down1.removePointer();
+
+    expect(focusNodeA.hasFocus, true);
+    expect(focusNodeB.hasFocus, false);
+
+    // Click on the container to not hit either text field.
+    final TestGesture down2 = await tester.startGesture(tester.getCenter(find.byKey(key)));
+    await tester.pump();
+    await tester.pumpAndSettle();
+    await down2.up();
+    await down2.removePointer();
+
+    expect(focusNodeA.hasFocus, false);
+    expect(focusNodeB.hasFocus, false);
+
+    // Second text field can still gain focus.
+
+    final TestGesture down3 = await tester.startGesture(tester.getCenter(find.byType(TextField).last));
+    await tester.pump();
+    await tester.pumpAndSettle();
+    await down3.up();
+    await down3.removePointer();
+
+    expect(focusNodeA.hasFocus, false);
+    expect(focusNodeB.hasFocus, true);
+  }, skip: !isBrowser); // https://github.com/flutter/flutter/issues/64245
```

Comment:
Is there anything else from your grid (https://github.com/flutter/flutter/pull/100347/#discussion_r873187233) that isn't already tested?

Reply:
Oh and I think you can remove the comment here.

Reply:
`A Focused text-field will lose focus when clicking outside of its hitbox with a mouse on desktop` has described the test in desktop. I added the same test for all browsers.